modules/mono/editor/mono_bottom_panel.cpp

```diff
@@ -63,7 +63,7 @@ void MonoBottomPanel::_update_build_tabs_list() {
 				item_tooltip += "Running";
 			}
 
-			if (!tab->build_exited || !tab->build_result == MonoBuildTab::RESULT_SUCCESS) {
+			if (!tab->build_exited || tab->build_result != MonoBuildTab::RESULT_SUCCESS) {
```

Comment:
Why not `== MonoBuildTab::RESULT_ERROR`? I ask since [I also noticed this](https://github.com/godotengine/godot/pull/22752/commits/4f7b33cdcfdcbc11bcc506018dff1b06db3cf3f6#diff-b6032b6412842fa0049c04ff023bcc7dL66).